service-13/meta.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "ST[13] large packet transfer",
+  "submodules" : [
+    {
+      "name": "Large packet downlink subservice",
+        "elements": [
+          {
+            "name": "Downlink processing",
+            "asn1Files": [
+              "LargePacketTransferMessageParts.acn",
+              "LargePacketTransferMessageParts.asn1",
+              "PUS-13-1.acn",
```

Comment:
I have just noticed, that current practice is not to include .acn files in this list. Are the omitted purposely?

Reply:
yes, as list is named "asn1FIles" :) 
in theory - current import wizard should (hopefully, if not, please report issue) find all .acn files matching .asn by name. It's a feature - to enable library potentially be used with multiple encodings

Reply:
Current wizard implementation does not support adding .acn files, so I will report the issue.